code/_onclick/item_attack.dm

```diff
@@ -24,7 +24,7 @@
 	return FALSE
 
 /obj/attackby(obj/item/I, mob/living/user, params)
-	return ..() || (can_be_hit && isitem(I) && I.attack_obj(src, user))
+	return ..() || (can_be_hit && I.attack_obj(src, user))
```

Comment:
This wasn't a very good way to stop a runtime. I'm almost positive `check_martial_counter` is the origin of the runtime, and even if it's not, doing an `isitem` check on something that should *only* be passed items is something we *want* to runtime, so we can see where the problem came from and not paper it over with a check like this.